Intersect.Client.Core/Interface/Game/Spells/SpellItem.cs

```diff
@@ -1,247 +1,289 @@
+using Intersect.Client.Core;
+using Intersect.Client.Framework.Content;
+using Intersect.Client.Framework.File_Management;
 using Intersect.Client.Framework.GenericClasses;
+using Intersect.Client.Framework.Gwen;
 using Intersect.Client.Framework.Gwen.Control;
 using Intersect.Client.Framework.Gwen.Control.EventArguments;
 using Intersect.Client.Framework.Gwen.Input;
 using Intersect.Client.Framework.Input;
 using Intersect.Client.General;
 using Intersect.Client.Interface.Game.DescriptionWindows;
+using Intersect.Client.Localization;
 using Intersect.Configuration;
 using Intersect.Framework.Core;
 using Intersect.GameObjects;
 using Intersect.Utilities;
 
 namespace Intersect.Client.Interface.Game.Spells;
 
-
-public partial class SpellItem
+public partial class SpellItem : SlotItem
 {
+    // Controls
+    private readonly Label _cooldownLabel;
+    private readonly SpellsWindow _spellWindow;
+    private Draggable? _dragIcon;
+    private SpellDescriptionWindow? _descWindow;
 
-    public ImagePanel Container;
-
+    // Drag Handling
     public bool IsDragging;
```

Comment:
This should be below the constructor and should be `{ get; private set; }` not a field